apps/agreement/test/helpers/wrappers/agreement.js

```diff
@@ -38,8 +38,9 @@ class AgreementWrapper {
   }
 
   async getChallenge(challengeId) {
-    const { actionId, context, endDate, challenger, settlementOffer, arbitratorFeeAmount, arbitratorFeeToken, state, disputeId, ruling, submitterFinishedEvidence, challengerFinishedEvidence } = await this.agreement.getChallenge(challengeId)
-    return { actionId, context, endDate, challenger, settlementOffer, arbitratorFeeAmount, arbitratorFeeToken, state, disputeId, ruling, submitterFinishedEvidence, challengerFinishedEvidence }
+    const { actionId, context, endDate, challenger, settlementOffer, state, disputeId, ruling, submitterFinishedEvidence, challengerFinishedEvidence } = await this.agreement.getChallenge(challengeId)
+    const { challengerArbitratorFeeAmount, challengerArbitratorFeeToken, submitterArbitratorFeeAmount, submitterArbitratorFeeToken }  = await this.agreement.getChallengeFees(challengeId)
+    return { actionId, context, endDate, challenger, settlementOffer, challengerArbitratorFeeAmount, challengerArbitratorFeeToken, submitterArbitratorFeeAmount, submitterArbitratorFeeToken, state, disputeId, ruling, submitterFinishedEvidence, challengerFinishedEvidence }
```

Comment:
Careful that this kind of changes increases a lot the test suite runtime, maybe it is better to add this a separate method instead of forcing every `getChallenge` to hit both functions